mi-scheduler/base/cronjob.yaml

```diff
@@ -2,11 +2,11 @@
 kind: CronJob
 apiVersion: batch/v1beta1
 metadata:
-  name: mi-scheduler
+  name: mi-scheduler-gh
```

Comment:
as this is udpate 
the name is to be updated in places in overlays
ex: https://github.com/thoth-station/thoth-application/blob/f095d33200b16b6119889669a77e28381a85d2bc/mi-scheduler/overlays/ocp4-stage/cronjob.yaml#L5

Reply:
done in all overlays